app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
```

Comment:
From the checklist:
If the function receives an incorrect type of data, it raises the correct exception.

Please cover this case as well